quickstart/latest/Fiddler_Quickstart_Ranking_Model.ipynb

```diff
@@ -306,154 +301,57 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATASET_NAME = 'expedia_dataset'\n",
-    "\n",
-    "sample_df_grouped = fdl.utils.helpers.group_by(df=sample_df, group_by_col='srch_id')\n",
+    "model = fdl.Model.from_data(\n",
+    "    name=MODEL_NAME,\n",
+    "    project_id=project.id,\n",
+    "    source=sample_data_df,\n",
+    "    spec=model_spec,\n",
+    "    task=model_task,\n",
+    "    task_params=task_params,\n",
+    "    event_ts_col=timestamp_column,\n",
+    ")\n",
     "\n",
-    "model.publish(\n",
-    "    source=sample_df_grouped,\n",
-    "    environment=fdl.EnvType.PRE_PRODUCTION,\n",
-    "    dataset_name=DATASET_NAME\n",
-    ")"
+    "try:\n",
```

Comment:
Seems odd that this is the only code we catch exceptions for.  To be more consistent, should this client method have a more graceful response if the project can't be created?

Reply:
I've added try/except to all init, create project, create model calls as these are the most frequent calls to throw stack traces. Could certainly take that farther.

Reply:
https://fiddler.productboard.com/all-notes/notes/43960376